httpretty_fixtures/__init__.py

```diff
@@ -96,6 +98,11 @@ def start(cls, fixtures):
             raise TypeError('Expected `fixtures` to be an iterable sequence but it was not. '
                             'Please make it a list or a tuple.')
 
+        # Keep track if HTTPretty was started outside of FixtureManager
+        #   This means that we should not auto-disable HTTPretty when nested_count returns to 0
+        if FixtureManager.nested_count == 0:
+            FixtureManager.disable_when_done = not HTTPretty.is_enabled()
```

Comment:
The logic seems sound but these negations are confusing me a bit. Can we rename the variables to something like:

```python
FixtureManager.httpretty_enabled_at_start = HTTPretty.is_enabled()
# ...
if FixtureManager.nested_count == 0 and FixtureManager.httpretty_enabled_at_start == False:
  HTTPretty.disable()
```